src/Features/CSharp/Portable/ConvertLinq/CSharpConvertLinqQueryToForEachProvider.cs

```diff
@@ -13,12 +15,12 @@
 using Microsoft.CodeAnalysis.LanguageServices;
 using Microsoft.CodeAnalysis.Operations;
 using Microsoft.CodeAnalysis.Shared.Extensions;
-using Microsoft.CodeAnalysis.Shared.Utilities;
 using Microsoft.CodeAnalysis.Simplification;
 using Microsoft.CodeAnalysis.Text;
 
 namespace Microsoft.CodeAnalysis.CSharp.ConvertLinq
 {
+    [ExportCodeRefactoringProvider(LanguageNames.CSharp, Name = nameof(CSharpConvertLinqQueryToForEachProvider)), Shared]
```

Comment:
We should consider writing a meta analyzer in Microsoft.CodeAnalysis.Analyzers to catch this. Can you please file a bug in analyzers repo?

Reply:
Note we already have such an analyzer/fixer for diagnostic analyzers that don't have `DiagnosticAnalyzerAttribute`.

Reply:
Thank you! Created https://github.com/dotnet/roslyn-analyzers/issues/1670